docs/sphinx-docs/build_sphinx.py

```diff
@@ -260,7 +261,9 @@ def build():
 
     print("=== Copy HTML Docs to Build Directory ===")
     html = joinpath(SPHINX_BUILD, "html")
+    copy_tree(SPHINX_SOURCE, SASVIEW_DOC_TARGET)
```

Comment:
This is a duplicate of line 250. Is there a reason for doing this a 2nd time? If there is, documenting why would be helpful.